include/class.ticket.php

```diff
@@ -2916,6 +2916,23 @@ function postReply($vars, &$errors, $alert=true, $claim=true) {
         if (isset($vars['ccs']) && count($vars['ccs']))
             $this->addCollaborators($vars['ccs'], array(), $errors);
 
+        if ($collabs = $this->getRecipients('collabs', array(), false)) {
```

Comment:
Why could you call 'getCollaborators' directly and avoid mailing list / contact mess?